ClosedXML/Excel/Cells/IXLCell.cs

```diff
@@ -310,8 +310,31 @@ public interface IXLCell
 
         IXLCell CopyTo(String target);
 
+        /// <summary>
+        /// Textual representation of cell calculated value (as it is saved to a workbook or read from it)
+        /// </summary>
         String ValueCached { get; }
 
+        /// <summary>
+        /// Calculated value of cell formula. Is used for decreasing number of computations perfromed.
+        /// May hold invalid value when <see cref="RecalculationNeeded"/> flag is True.
+        /// </summary>
+        Object ValueCalculated { get; }
+
+        /// <summary>
+        /// Flag indicating that previously calculated cell value may be not valid anymore and has to be re-evaluated.
+        /// </summary>
+        Boolean RecalculationNeeded { get; }
```

Comment:
Please change to `NeedsRecalculation`. I prefer `VerbObject` format (active voice) for property names, instead of passive voice.